drivers/clock_control/clock_stm32f0_f3.c

```diff
@@ -16,7 +16,7 @@
 #include "clock_stm32_ll_common.h"
 
 
-#ifdef CONFIG_CLOCK_STM32_SYSCLK_SRC_PLL
+#if CONFIG_SOC_STM32_SYSCLK_SRC_PLL
```

Comment:
`CONFIG_CLOCK_` prefix was removed in previous changes, why a `CONFIG_SOC_` prefix here ?

Reply:
Indeed. Since I'm not changing F0/F3 (and L0/L1 ...), I've preferred to limit the changes in those files and reverted few of the changes. They will be done in later PR once this one is approved.

Reply:
ok